homeassistant/components/smartthings/cover.py

```diff
@@ -162,7 +163,7 @@ def current_cover_position(self) -> int | None:
         """Return current position of cover."""
         return self._current_cover_position
 
-    @property
+    @cached_property
```

Comment:
This one is set via `async_update` but thats ok since the entity is added with `update_before_add=True`

We could set the shorthand in `async_update` instead